core/src/main/scala/org/apache/spark/SparkThrowableHelper.scala

```diff
@@ -119,4 +121,16 @@ private[spark] object SparkThrowableHelper {
         }
     }
   }
+
+  def getMessage(throwable: Throwable): String = {
+    toJsonString { generator =>
+      val g = generator.setPrettyPrinter(new MinimalPrettyPrinter)
+      g.writeStartObject()
+      g.writeStringField("errorClass", throwable.getClass.getCanonicalName)
+      g.writeObjectFieldStart("messageParameters")
```

Comment:
Could you share a screenshot of the error message? This line looks odd to me: it just writes a constant string